core-services/prow/02_config/_config.yaml

```diff
@@ -1,3 +1,13 @@
+slack_reporter_configs:
+  "*":
+    job_types_to_report:
+    - presubmit
+    - postsubmit
+    - periodic
+    - batch
+    job_states_to_report:
+    - error
+    channel: "#ops-testplatform"
```

Comment:
What? That will be like a billion reports per second

Reply:
```
    job_states_to_report:
    - error
```

Reply:
we should only see the jobs that fail from infra reasons

Reply:
Okay thats not as bad but still, if we have an outage, it will spam like crazy. Wouldn't it make more sense to have a metric for this?

Reply:
Yeah, true. I think we'll need to re-work the slack config to change how it works since reporting just one job is not possible

Reply:
Actually, I think we can just reconfigure